checkups/kubevirt-vm-latency/vmlatency/internal/vmi/spec.go

```diff
@@ -207,6 +210,25 @@ func WithMacAddress(macAddress string) interfaceOption {
 	}
 }
 
+// RandomMACAddress return random local unicast MAC address
+func RandomMACAddress() string {
+	const octetsCount = 6
+	address := make([]byte, octetsCount)
+	_, _ = rand.Read(address)
+	address[0] = 0x02
```

Comment:
In this byte we only care that the least significant bits are 10 (locally managed address and unicast).
The rest of the bits could be random.

```suggestion
	address[0] = (address[0] | 2) & 0xFE
```

Reply:
I think that setting 0x02 directly is easier to understand than what you suggest, there are many ways to solve this problem lets keep it simple as possible.